front/src/modules/ui/table/components/EntityTableBody.tsx

```diff
@@ -50,7 +50,7 @@ export const EntityTableBody = () => {
   }
 
   return (
-    <tbody>
+    <tbody data-testid="t-body">
```

Comment:
Following https://github.com/twentyhq/twenty/pull/1714/files#r1337264035, the `data-testid` here wouldn't be needed anymore!